open-apparel-registry/open-apparel-registry
# Create allow large downloads flag (#2081)

## Overview

Creates an allow large downloads flag, which if not set to true limits downloads to result sets smaller than 10000 facilities.

Connects #2081

## Testing Instructions
* Run migrations so you can get the new switch
* Because test fixtures seed 901 facilities, you will need to lower the limit here, e.g. to 100:
  * `src/app/src/components/FilterSidebarFacilitiesTab.jsx:230`
* Searching without filters will lead to the download button being disabled and showing a tooltip
* Searching with a filter like "socks" which filters the results down below the threshold should enable the download button again
* Log in to the django admin console and go to http://localhost:8081/admin/waffle/switch/ and activate the switch to allow large downloads
* The download button should be enabled and the tooltip should be missing on every search now

## Checklist
